src/cargo/util/command_prelude.rs

```diff
@@ -90,7 +90,10 @@ pub trait CommandExt: Sized {
         self._arg(
             optional_multi_opt("package", "SPEC", package)
                 .short('p')
-                .help_heading(heading::PACKAGE_SELECTION),
+                .help_heading(heading::PACKAGE_SELECTION)
+                .add(clap_complete::ArgValueCandidates::new(
+                    get_ws_member_candidates,
+                )),
```

Comment:
Looks like `cargo uninstall` uses this flag for the same reason as the positional argment

Reply:
`arg_package_spec_no_all(` calls this
- `cargo tree` uses to refer to any package in the dependency graph

`arg_package_spec` calls that but all uses look good